### PR TITLE
[README] document ROS_PACKAGE_PATH use

### PR DESCRIPTION
**Pull request by nim65s**
While here, and because I have plenty of time because everything is compiling everywhere, I simplified the readme. Here is a rendering: https://github.com/nim65s/example-robot-data/blob/topic/readme/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,52 +1,58 @@
...
-		deb [arch=amd64] http://robotpkg.openrobots.org/packages/debian/pub $(lsb_release -sc) robotpkg
-		EOF
+   curl http://robotpkg.openrobots.org/packages/debian/robotpkg.key | sudo apt-key add -
+   sudo apt update
+   ```
 
-2. Register the authentication certificate of robotpkg:
+2. installation of example-robot-data and its python utils:
+   ```bash
+   sudo apt install robotpkg-py3\*-example-robot-data
```
Comment (cmastalli) [inline README.md:33]: There is a typo here

File: README.md [hunk 2]
```diff
@@ -1,52 +1,58 @@
...
+   ```
 
-		curl http://robotpkg.openrobots.org/packages/debian/robotpkg.key | sudo apt-key add -
+3. If you install this in a non-standard directory and **without catkin**, you will need to set `ROS_PACKAGE_PATH` to
+   your `$CMAKE_INSTALL_PREFIX/share` (`/usr/local/share` by default)
```
Comment (wxmerkt) [inline README.md:37]: Does this mean that robotpkg does *not* install to `/opt/openrobots/share` but to `/usr/local/share`?

File: README.md (hunk 1 above)
Comment (nim65s) [reply README.md:33]: If I copy-paste that, it works, no matter in you are in 16.04 or 18.04 and have python 3.5 or 3.6.

File: README.md (hunk 2 above)
Comment (nim65s) [reply README.md:37]: `/usr/local` is the default `CMAKE_INSTALL_PREFIX. But you are right, we should talk about robotpkg.

**Comment by nim65s** [comment]
@cmastalli : I had a website, but it is currently down. I don't have time for it right now, but I'll setup one some day, and add it here.

File: README.md (hunk 1 above)
Comment (cmastalli) [reply README.md:33]: I was thinking in Python 2.5

File: README.md (hunk 2 above)
Comment (wxmerkt) [reply README.md:37]: I think we should also add as prominently as a second installation option that simply cloning this into a catkin workspace works just fine, no further configuration needed.

The /usr/local should be option 3 - pure CMake locally.

Comment (nim65s) [reply README.md]: I actually wrote the "install from source" part as you described, and it took a lot of place for only standard boring stuff, so I removed it. What do you think of the current state ? We could add a catkin part, especially if this is really short :)

**Comment by nim65s** [comment]
Hi,

I'd like to make a release with this.
Is https://github.com/nim65s/example-robot-data/blob/topic/readme/README.md OK for you, @cmastalli & @wxmerkt ?

**Comment by cmastalli** [comment]
@nim65s I approve these changes. Thank you for taking care of this :)

**Comment by wxmerkt** [comment]
Good for me, thank you!

**Pull request merged**